ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -363,7 +379,9 @@ tests:
   optional: true
   steps:
     cluster_profile: packet
```

Comment:
Need to change to `cluster_profile: packet-assisted` since we are making changes here.